components/global/user_guide_dropdown/user_guide_dropdown.tsx

```diff
@@ -0,0 +1,130 @@
+        e.preventDefault();
+        toggleShortcutsModal();
+    }
+
+    buttonToggleState = (menuActive: boolean) => {
+        this.setState({
+            buttonActive: menuActive,
+        });
+    }
+
+    askTheCommunityClick = () => {
+        trackEvent('ui', 'help_ask_the_community');
+    }
+
+    renderDropdownItems = (): React.ReactNode => {
+        const {intl} = this.props;
+
+        return (
+            <Menu.Group>
+                {this.props.enableAskCommunityLink === 'true' && (
+                    <Menu.ItemExternalLink
+                        id='askTheCommunityLink'
+                        url={askTheCommunityUrl}
+                        text={intl.formatMessage({id: 'userGuideHelp.askTheCommunity', defaultMessage: 'Ask the community'})}
+                        onClick={this.askTheCommunityClick}
+                    />
+                )}
+                <Menu.ItemExternalLink
+                    id='helpResourcesLink'
+                    url={this.props.helpLink}
+                    text={intl.formatMessage({id: 'userGuideHelp.helpResources', defaultMessage: 'Help resources'})}
+                />
+                <Menu.ItemExternalLink
+                    id='reportAProblemLink'
+                    url={this.props.reportAProblemLink}
+                    text={intl.formatMessage({id: 'userGuideHelp.reportAProblem', defaultMessage: 'Report a problem'})}
+                />
+                <Menu.ItemAction
+                    id='keyboardShortcuts'
+                    onClick={this.toggleShortcutsModal}
+                    text={intl.formatMessage({id: 'userGuideHelp.keyboardShortcuts', defaultMessage: 'Keyboard shortcuts'})}
+                />
+            </Menu.Group>
+        );
+    }
+
+    render() {
+        const {intl} = this.props;
+        const tooltip = (
+            <Tooltip
+                id='userGuideHelpTooltip'
+                className='hidden-xs'
+            >
+                <FormattedMessage
+                    id={'channel_header.userHelpGuide'}
+                    defaultMessage='Help'
+                />
+            </Tooltip>
+        );
+
+        return (
+            <MenuWrapper
+                className='userGuideHelp'
+                onToggle={this.buttonToggleState}
+            >
+                <OverlayTrigger
+                    delayShow={500}
+                    placement='bottom'
+                    overlay={this.state.buttonActive ? <></> : tooltip}
+                >
+                    <IconButton
+                        size={'sm'}
+                        icon={'help-circle-outline'}
+                        onClick={() => {}} // icon button currently requires onclick ... needs to revisit
+                        toggled={this.state.buttonActive}
+                        inverted={true}
+                        compact={true}
+                        aria-label='Select to toggle the help menu.' // proper wording and translation needed
```

Comment:
Should we not handle this now?

Reply:
We can remove it but its good to keep it as a reminder IMO

Reply:
One step at a time, this is a placeholder of additional work that still needs to be done.

I created a placeholder ticket: https://mattermost.atlassian.net/browse/MM-37792

Reply:
A ticket is a great reminder to do something later.

Reply:
all these tickets are in our active sprint so we can't avoid it :D